### PR TITLE
Notesnook v3.0.8 -> v3.0.13

### PR DESCRIPTION
**Pull request by Marin-Kitagawa**
This is an update only for x86-64 Linux. Version for other platforms (v3.0.8) remains unchanged for other platforms.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/misc/notesnook/default.nix [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-  version = "3.0.8";
-
   inherit (stdenv.hostPlatform) system;
   throwSystem = throw "Unsupported system: ${system}";
 
+  pname = "notesnook";
+  version = {
+    x86_64-linux = "3.0.13";
+    x86_64-darwin = "3.0.8";
+    aarch64-darwin = "3.0.8";
```
Comment (xzfc) [inline pkgs/applications/misc/notesnook/default.nix:11]: Why `x86_64-linux` only but not the others?

Comment (Marin-Kitagawa) [reply pkgs/applications/misc/notesnook/default.nix:11]: Hey @xzfc,
I actually don't own a Mac or ARM-based Linux system. So, I couldn't test them. I am new to NixOS and I don't know any other way to test them without actual MacBook or ARM-based Linux system

Comment (xzfc) [reply pkgs/applications/misc/notesnook/default.nix:11]: We rarely use diverging versions of the same package for different platforms. As per [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#built-on-platforms), it's not required to test such changes on all platforms.

So, feel free to bump it for all platforms. Someone will check it sooner or later. To find hashes for other platforms you could use the command `nix store prefetch-file`.

Comment (Marin-Kitagawa) [reply pkgs/applications/misc/notesnook/default.nix:11]: Oh sure. Will do that

Comment (Marin-Kitagawa) [reply pkgs/applications/misc/notesnook/default.nix]: Done

**Review by cig0 — APPROVED** [review]
LGTM

File: pkgs/applications/misc/notesnook/default.nix [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
-  pname = "notesnook";
-  version = "3.0.8";
-
   inherit (stdenv.hostPlatform) system;
   throwSystem = throw "Unsupported system: ${system}";
 
+  pname = "notesnook";
+  version = "3.0.13";
+
+
```
Comment (xzfc) [inline pkgs/applications/misc/notesnook/default.nix:10]: Could you please put these lines back to the top?

And please, squash your commits into a single one named `notesnook: 3.0.8 -> 3.0.13`, to follow the [commit conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions) for packages.

Comment (Marin-Kitagawa) [reply pkgs/applications/misc/notesnook/default.nix]: Yeah done

Comment (xzfc) [reply pkgs/applications/misc/notesnook/default.nix]: That's better but you also need to remove descriptions of squashed commits.

Comment (Marin-Kitagawa) [reply pkgs/applications/misc/notesnook/default.nix]: Check now

**Comment by Marin-Kitagawa** [comment]
Hey @cig0 could you please review this?

**Review by pbsds — COMMENTED** [review]
please add aarch64-linux to  `meta.platforms` and resolve the merge conflict

**Pull request closed without merging**